modules/gltf/src/index.ts

```diff
@@ -42,6 +42,7 @@ export {GLBWriter} from './glb-writer';
 // glTF Data Access Helper Class
 export {default as GLTFScenegraph} from './lib/api/gltf-scenegraph';
 export {postProcessGLTF} from './lib/api/post-process-gltf';
+export {getMemoryUsageGLTF} from './lib/gltf-utils/gltf-utils';
```

Comment:
Nit: This should probably be an underscore export to mark it as internal / not audited.